fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/FindInPageRobot.kt

```diff
@@ -27,49 +27,76 @@ import org.mozilla.fenix.helpers.ext.waitNotNull
+        Log.i(TAG, "verifyFindInPageCloseButton: Verified find in page close button is visible")
+    }
     fun clickFindInPageNextButton() {
+        Log.i(TAG, "clickFindInPageNextButton: Trying to click next result button")
         findInPageNextButton().click()
         Log.i(TAG, "clickFindInPageNextButton: Clicked next result button")
     }
     fun clickFindInPagePrevButton() {
+        Log.i(TAG, "clickFindInPagePrevButton: Trying to click previous result button")
         findInPagePrevButton().click()
         Log.i(TAG, "clickFindInPagePrevButton: Clicked previous result button")
     }
 
     fun enterFindInPageQuery(expectedText: String) {
         mDevice.waitNotNull(Until.findObject(By.res("org.mozilla.fenix.debug:id/find_in_page_query_text")), waitingTime)
+        Log.i(TAG, "enterFindInPageQuery: Trying to cleared find in page bar text")
         findInPageQuery().perform(clearText())
         Log.i(TAG, "enterFindInPageQuery: Cleared find in page bar text")
         mDevice.waitNotNull(Until.gone(By.res("org.mozilla.fenix.debug:id/find_in_page_result_text")), waitingTime)
+        Log.i(TAG, "enterFindInPageQuery: Trying to type $expectedText to find in page bar")
         findInPageQuery().perform(typeText(expectedText))
         Log.i(TAG, "enterFindInPageQuery: Typed $expectedText to find in page bar")
         mDevice.waitNotNull(Until.findObject(By.res("org.mozilla.fenix.debug:id/find_in_page_result_text")), waitingTime)
     }
 
     fun verifyFindInPageResult(ratioCounter: String) {
         mDevice.waitNotNull(Until.findObject(By.text(ratioCounter)), waitingTime)
+        Log.i(TAG, "verifyFindInPageResult: Trying to verify $ratioCounter results")
         findInPageResult().check(matches(withText((ratioCounter))))
         Log.i(TAG, "verifyFindInPageResult: Verified $ratioCounter results")
     }
 
     class Transition {
         fun closeFindInPageWithCloseButton(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
+            Log.i(TAG, "closeFindInPageWithCloseButton: Waiting for device to be idle for $waitingTime ms")
             mDevice.waitForIdle()
+            Log.i(TAG, "closeFindInPageWithCloseButton: Device was idle for $waitingTime ms")
+            Log.i(TAG, "closeFindInPageWithCloseButton: Trying to close find in page button")
             findInPageCloseButton().click()
             Log.i(TAG, "closeFindInPageWithCloseButton: Clicked close find in page button")
             BrowserRobot().interact()
             return BrowserRobot.Transition()
         }
 
         fun closeFindInPageWithBackButton(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
+            Log.i(TAG, "closeFindInPageWithBackButton: Waiting for device to be idle for $waitingTime ms")
             mDevice.waitForIdle()
+            Log.i(TAG, "closeFindInPageWithBackButton: Device was idle for $waitingTime ms")
```

Comment:
The default wait time for this is 10 seconds, unless specified otherwise (there is a parametrized version of this method `waitForIdle(long timeout)`. So 'Device was idle for $waitingTime' is incorrect, and also, we can't really know how long it took to be idle. We only know how long we can wait for it to be.

Reply:
yes, true, my bad, will change the phrasing.
thanks !